### DISCUSSION TITLE
TShock 5.0.0 Auto-close on launch with an error

### DISCUSSION THREAD

**Discussion by yokozuna30** [Pryaxis/TShock#2806] [General]
Hi. My TShock is closing automaticaly on launch with some kind of an error.
Windows 10 x64, already installed .NET 6 (desktop version). I'm not sure what to do, can someone help me, please?

![image](https://user-images.githubusercontent.com/117522582/200121844-c5f5f3d4-a6f9-412f-a19f-f9515dce5027.png)

**Comment by Arthri** [comment]
Do you have OTAPI.dll in your bin folder? It comes with TShock by default

**Reply by yokozuna30** [reply]
Yes

**Reply by yokozuna30** [reply]
I reinstalled game completely, and looks like error is gone. Anyway - thanks!
